11/umbraco-cms/reference/notifications/README.md

```diff
@@ -175,31 +178,31 @@ public class NotificationHandlersComposer : IComposer
-* See [MemberService Notifications](memberservice-notifications.md) for a listing of the MemberService object notifications.
+* See [ContentService Notifications](contentservice-notifications.md) for a list of the ContentService object notifications.
+* See [MediaService Notifications](mediaservice-notifications.md) for a list of the MediaService object notifications.
+* See [MemberService Notifications](memberservice-notifications.md) for a list of the MemberService object notifications.
 
 ## Other notifications
 
-* See [ContentTypeService Notifications](contentypeservice-notifications.md) for a listing of the ContentTypeService object notifications.
-* See [MediaTypeService Notifications](mediatypeservice-notifications.md) for a listing of the MediaTypeService object notifications.
-* See [MemberTypeService Notifications](membertypeservice-notifications.md) for a listing of the MemberTypeService object notifications.
-* See [DataTypeService Notifications](datatypeservice-notifications.md) for a listing of the DataTypeService object notifications
-* See [FileService Notifications](fileservice-notifications.md) for a listing of the FileService object notifications.
-* See [LocalizationService Notifications](localizationservice-notifications.md) for a listing of the LocalizationService object notifications.
+* See [ContentTypeService Notifications](contentypeservice-notifications.md) for a list of the ContentTypeService object notifications.
+* See [MediaTypeService Notifications](mediatypeservice-notifications.md) for a list of the MediaTypeService object notifications.
+* See [MemberTypeService Notifications](membertypeservice-notifications.md) for a list of the MemberTypeService object notifications.
+* See [DataTypeService Notifications](datatypeservice-notifications.md) for a list of the DataTypeService object notifications.
+* See [FileService Notifications](fileservice-notifications.md) for a list of the FileService object notifications.
+* See [LocalizationService Notifications](localizationservice-notifications.md) for a list of the LocalizationService object notifications.
 
 ## Tree notifications
 
-See [Tree Notifications](../../extending/section-trees/) for a listing of the tree notifications.
+See [Tree Notifications](../../extending/section-trees/) for a list of the tree notifications.
 
 ## Editor Model Notifications
 
-See [EditorModel Notifications](editormodel-notifications/) for a listing of the EditorModel events
+See [EditorModel Notifications](editormodel-notifications/) for a list of the EditorModel events
 
 {% hint style="info" %}
-Useful for manipulating the model before it is sent to an editor in the backoffice - e.g. perhaps to set a default value of a property on a new document.
+Useful for manipulating the model before it is sent to an editor in the backoffice - e.g. to set a default value of a property on a new document.
```

Comment:
Another suggestion to shorten/split up a long sentence.

```suggestion
Useful for manipulating the model before it is sent to an editor in the backoffice. It could be used to set a default value of a property on a new document.
```